### PR TITLE
Installing maps-4.2.0 for unittest

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/commons#810

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @cloudbehl please review

File: setup.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def run(self):
         "python-etcd",
         "python-dateutil",
         "ruamel.yaml",
-        "maps",
+        "maps==4.2.0",
```
Comment (shtripat) [inline setup.py]: Have you tested with change and do the tests pass now?

**Comment by r0h4n** [comment]
Please fix failing unit tests in this PR

**Comment by GowthamShanmugam** [comment]
I will fix unit test cases also

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n I have fixed unit test cases in common

**Pull request merged**